delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchPlugin.scala

```diff
@@ -0,0 +1,22 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch
+
+import akka.http.scaladsl.server.Route
+import ch.epfl.bluebrain.nexus.delta.sdk.plugin.{Plugin, PluginInfo}
+import monix.bio.Task
+
+class ElasticSearchPlugin extends Plugin {
+  /**
+    * Plugin information
+    */
+  override def info: PluginInfo = ???
+
+  /**
+    * Optional routes provided by the plugin.
+    */
+  override def route: Option[Route] = ???
```

Comment:
We would also need some sort of priority? When we assemble the routes in the core, we should probably pick up all the routes from plugins and core in something like:
```scala
val routes: List[(Priority, Route)] = ???
va sortedRoutes = routes.sortBy{ case (prio, _) => prio }.map { case (_, route) => route}
```

Reply:
Alternatively, the priority  information can also be in `PluginInfo`

Reply:
that could possibly be useful, but it's not in the scope of this PR... I've just implemented the plugin contract with `???`